<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>70</y>
      <width>281</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文行楷</family>
      <pointsize>59</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startBtn1">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>280</y>
      <width>281</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文行楷</family>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string>打乱并开始</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>新建</string>
    </property>
    <addaction name="actionthreeorder"/>
    <addaction name="actionfourorder"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="actionReset"/>
    <addaction name="actionUpset"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>公式</string>
    </property>
    <addaction name="debugMode"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="actionthreeorder">
   <property name="text">
    <string>三阶</string>
   </property>
  </action>
  <action name="actionfourorder">
   <property name="text">
    <string>四阶</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>重置</string>
   </property>
  </action>
  <action name="actionUpset">
   <property name="text">
    <string>打乱</string>
   </property>
  </action>
  <action name="debugMode">
   <property name="text">
    <string>调试模式</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
